src/main/java/frc/robot/Constants.java

```diff
@@ -117,6 +119,35 @@ public static final class RobotConstants {
     public static final double kFrontRightChassisAngularOffset = 0;
     public static final double kBackLeftChassisAngularOffset = Math.PI;
     public static final double kBackRightChassisAngularOffset = Math.PI / 2;
+
+    // Algae Arm gear ratio, 15 to 1, so motor rotations to arm rotations is 1 to 15
+    public static final double kAlgaeArmGearRatio = 1.0 / 15.0;
+
+    // Array of heights of algae arm stops relative to base height
+    // Indicies are stops in order from bottom to top
+    public static final double[] kAlgaeArmStops = {
+      RobotConstants.kAlgaeArmBottomAngle,
+      RobotConstants.kAlgaeArmFlatAngle,
+      RobotConstants.kAlgaeArmRemoveAlgaeAngle,
+      RobotConstants.kAlgaeArmTopAngle
+    };
+
+    // Angles for Algae Arm in Degrees are: -60, 0, 40, 60
+    // Following Angles are in rotations relative to bottom.
+    public static final double kAlgaeArmBottomAngle = 0;
+    public static final double kAlgaeArmFlatAngle = 1.0 / 6.0;
```

Comment:
Should change so that angles are provided in constants and conversion happens elsewhere. Makes it easier to change from an external programmer's perspective